grid/social-media/profiles/maps/twitter.suma

```diff
@@ -51,6 +51,83 @@ map GetProfiles {
   }
 }
 
+map GetProfilesByUsername {
+  // https://developer.twitter.com/en/docs/twitter-api/users/lookup/api-reference/get-users-by
+  http GET "/2/users/by" {
+    // FIXME: support pagination by checking if usernames.length > 100 and providing remaining IDs in nextPage
+    request {
+      query {
+        usernames = input.usernames.join(','),
+        'user.fields' = 'id,username,name,description,profile_image_url,url,location,public_metrics'
+      }
+      headers {
+        authorization = "Bearer " + parameters.accessToken
+      }
+    }
+
+    response 200 "application/json" {
+      profiles = body.data.map((user) => {
```

Comment:
I suggest to map user in operation `MapUser`. Mapping code is exactly the same for both use-cases.

Reply:
Yes, I thought the same, I just postponed this in favor of delivery speed. I will merge this and send a separate PR for this.